standard_reports/item-usage/latest/convert_data_js/src/test/test.ts

```diff
@@ -0,0 +1,11 @@
+import { convert_data } from '../convert_data';
+import inputData from './input.json';
+import outputData from './output.json';
+import { describe, expect, it } from '@jest/globals';
+
+describe('Test all', () => {
+  it('test all', () => {
+    const result = convert_data(inputData);
```

Comment:
I think we need more 'overall' test for reports, rather then testing individual methods (unless overall fail and you need to drill down on the error)